Makefile

```diff
@@ -21,9 +21,9 @@ venv: requirements.txt
 	venv/bin/pip install -r requirements.txt
 
 test: venv FORCE
-	venv/bin/pep8 .
-	venv/bin/yamllint .
 	venv/bin/nosetests --with-coverage --cover-package=.
+	venv/bin/pep8 . --exclude venv --max-line-length 120
```

Comment:
By default, `pep8` fails our automated code review with any line over 80 characters. That's a great default for running on a VT100, but most folks have higher resolution screens these days, so we change it to only complaining of lines over 120 characters.

Having said that, this is a regression caused by me failing to resolve a merge conflict properly. All our pep8 settings now live in `setup.cfg`, so they get used regardless of how they run. (New commit to fix this incoming.)